tests/arp/test_neighbor_mac.py

```diff
@@ -35,15 +36,40 @@ def interfaceConfig(self, duthosts, rand_one_dut_hostname):
                 None
         """
         duthost = duthosts[rand_one_dut_hostname]
-        logger.info("Configure the DUT interface, start interface, add IP address")
-        self.__startInterface(duthost)
-        self.__configureInterfaceIp(duthost, action="add")
 
-        yield
-
-        logger.info("Restore the DUT interface config, remove IP address")
-        self.__configureInterfaceIp(duthost, action="remove")
-        self.__shutdownInterface(duthost)
+        intfStatus = duthost.show_interface(command="status")["ansible_facts"]["int_status"]
+        if self.DUT_ETH_IF not in intfStatus:
+            pytest.skip('{} not found'.format(self.DUT_ETH_IF))
+
+        status = intfStatus[self.DUT_ETH_IF]
+        if "up" not in status["oper_state"]:
+            pytest.skip('{} is down'.format(self.DUT_ETH_IF))
+
+        portchannel = status["vlan"] if "PortChannel" in status["vlan"] else None
+
+        @contextlib.contextmanager
+        def removeFromPortChannel(duthost, portchannel, intf):
+            try:
+                if portchannel:
+                    duthost.command("sudo config portchannel member del {} {}".format(portchannel, intf))
+                    time.sleep(2)
+                    intfStatus = duthost.show_interface(command="status")["ansible_facts"]["int_status"]
```

Comment:
better to add a `wait_until` logic here to wait and check.

Reply:
Done